WPMedia/arc-themes-components
# Update Storybook & Fix Docs loading error

## Description

* Updated storybook to latest version
* Included the ESLint storybook addon as recommend by their cli tool during upgrading
* Fixed the issue where you can not navigate directly to a docs page - http://localhost:60003/?path=/docs/components-heading--heading


## Test Steps

1. Checkout branch - `git checkout fix-storybook-loading-docs-issue`
2. Update dependencies - `npm i`
3. Run Storybook `npm run storybook`
4. Check out the button storybook documentation
5. Validate the following URL loads - http://localhost:60003/?path=/docs/components-heading--heading

Also Chromatic URL's load as intended now - https://61eeeb27215b5f003acdf18f-fvpkvnsdxa.chromatic.com/?path=/docs/components-button--button

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
